### PR TITLE
Molecule progress steps

### PR DESCRIPTION
**Pull request by juanmaguitar**
- ⚙️ Task: https://jira.scmspain.com/browse/SUIC-47
- 📄 Paper: https://paper.dropbox.com/doc/SUI-Progress-steps-WIP--AO4LRxTAMb0jymT51xjQYBCMAQ-EzeFBpQuZZAXFJVoMBpmn
- 🌍 Demo Online: https://sui-components-molecule-progress-steps.now.sh/workbench/molecule/progressSteps/demo

✅ OK por parte de UX → https://jira.scmspain.com/browse/SUIC-47?focusedCommentId=787554&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-787554

### CODE REVIEW

File: components/molecule/progressSteps/src/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  iconStepDone: PropTypes.node.isRequired,
+
+  /** Compressed mode (mobile) */
+  compressed: PropTypes.bool,
+
+  /** Vertical mode */
+  vertical: PropTypes.bool
+}
+
+// MoleculeProgressSteps.defaultProps = {}
```
Comment (ddmarin94) [inline components/molecule/progressSteps/src/index.js]: 👀

File: components/molecule/progressSteps/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,146 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-progress-steps-bar: #bbbbbb !default;
```
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply components/molecule/progressSteps/src/index.scss]: Fixed!

File: components/molecule/progressSteps/src/components/MoleculeProgressStep/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+const MoleculeProgressStep = ({
+  status,
+  icon,
+  label,
+  numStep,
+  lastStep,
+  compressed,
+  children
```
Comment (rmoralp) [inline components/molecule/progressSteps/src/components/MoleculeProgressStep/index.js]: `Children` is not used here.

File: components/molecule/progressSteps/src/components/MoleculeProgressStep/index.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+          <p className={CLASS_STEP_DESCRIPTION}>{label}</p>
+        </div>
+      )}
+      {!lastStep ? bar : compressed && bar}
+    </Fragment>
+  )
+}
+
+MoleculeProgressStep.propTypes = {
+  /** children */
```
Comment (rmoralp) [inline components/molecule/progressSteps/src/components/MoleculeProgressStep/index.js]: Could you specify somewhere that this children is used only in `MoleculeProgressSteps` component?

**Review by rmoralp — DISMISSED** [review]

**Review by rmoralp — APPROVED** [review]

**Review by danivm — APPROVED** [review]

**Pull request merged**